infinistore/server.py

```diff
@@ -7,6 +7,10 @@
 import torch
 import argparse
 
+import logging
+
+logging.disable(logging.INFO)
```

Comment:
not totally disable uvicorn's log.  uvicorn should  output log using log.cpp's log instead of python log.

Reply:
uvicorn also uses logging for log, so you still want the uvicorn log but printing the log with log.cpp?
how about changing the py log format to the same as lop.cpp?